x2gbfs/providers/cantamen.py

```diff
@@ -174,12 +174,16 @@ def _seats(self, attributes: List[str]) -> Optional[int]:
     def _filter_and_map_attributes(self, attributes: List[str], filter: List[str]) -> List[str]:
         filtered_attributes = []
         for attribute in attributes:
+            if self.attributes.get(attribute) is None:  # cantamen data error
+                continue
             if self.attributes[attribute] in filter:
```

Comment:
I suggest to combine this with the following if, instead of accessing via [], get() will return None, the condition will not be true and the loop be continued also.